flask_resources/resources.py

```diff
@@ -24,23 +24,54 @@
 class ResourceConfig:
     """Base resource configuration."""
 
+    # Existing
     request_loaders = {
-        "application/json": RequestLoader(
-            deserializer=JSONDeserializer(), args_parser=search_request_parser,
-        )
+        "application/json": RequestLoader(deserializer=JSONDeserializer())
     }
     response_handlers = {"application/json": Response(JSONSerializer())}
     item_route = "/resources/<id>"
     list_route = "/resources/"
 
+    # Suggested
+    request_url_args_parser = ArgsParser()
+    # incoming = {
```

Comment:
major: since this is a draft/suggestion and there is no consensus... can we move out this part, and config related, to another PR...

So it does not block merging the args parser

Reply:
You mean removing the comments? I can do that.